metricbeat/mb/testing/testdata.go

```diff
@@ -42,7 +43,7 @@ import (
 
 const (
 	expectedExtension  = "-expected.json"
-	applicationJson    = "application/json"
+	textPlain          = "text/plain"
```

Comment:
Nice! Maybe a link why we did this change for someone to understand what we test here